### PR TITLE
#134 Add CLI logic options -s Tax Categories

### PR DESCRIPTION
**Pull request by leungkinghin**
Fix Issue #134

### CODE REVIEW

File: src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java [hunk 1]
```diff
@@ -120,6 +130,22 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
 
     sourceProjectClient.execute(TypeCreateCommand.of(typeDraft)).toCompletableFuture().join();
 
+    final TaxRateDraft draft =
```
Comment (ahmetoz) [inline src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java:133]: a small refactor for list creation:
```
final TaxRateDraft taxRateDraft =
        TaxRateDraftBuilder.of("Tax-Rate-Name-1", 0.3, false, CountryCode.DE).build();

    final TaxCategoryDraft taxCategoryDraft =
        TaxCategoryDraftBuilder.of(
                "Tax-Category-Name-1", singletonList(taxRateDraft), "Tax-Category-Name-1")
            .key(RESOURCE_KEY)
            .build();
```

File: src/test/java/com/commercetools/project/sync/CliRunnerTest.java [hunk 2]
```diff
@@ -195,6 +196,32 @@ void run_WithSyncAsArgumentWithNoArgs_ShouldFailAndLogError() {
             });
   }
 
+  @Test
```
Comment (ahmetoz) [inline src/test/java/com/commercetools/project/sync/CliRunnerTest.java:199]: The order of the functions looks weird, maybe put it both tests after `run_AsProductFullSync_ShouldBuildSyncerAndExecuteSync` not middle of product delta and product full sync test.

File: src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java (hunk 1 above)
Comment (leungkinghin) [reply src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java]: https://github.com/commercetools/commercetools-project-sync/commit/f4f354eb8092d285d2fdef52efaf669efb908efe

File: src/test/java/com/commercetools/project/sync/CliRunnerTest.java (hunk 2 above)
Comment (leungkinghin) [reply src/test/java/com/commercetools/project/sync/CliRunnerTest.java]: https://github.com/commercetools/commercetools-project-sync/commit/546fdeb0831dccb3ee1d3823e23df2afbfa54b34

File: src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java [hunk 3]
```diff
@@ -120,6 +129,21 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
 
     sourceProjectClient.execute(TypeCreateCommand.of(typeDraft)).toCompletableFuture().join();
 
+    final TaxRateDraft taxRateDraft =
+        TaxRateDraftBuilder.of("Tax-Rate-Name-1", 0.3, false, CountryCode.DE).build();
+
+    final TaxCategoryDraft taxCategoryDraft =
+        TaxCategoryDraftBuilder.of(
+                "Tax-Category-Name-1", singletonList(taxRateDraft), "Tax-Category-Name-1")
```
Comment (JudeNiroshan) [inline src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java:137]:
```suggestion
                "Tax-Category-Name-1", singletonList(taxRateDraft), "Tax-Category-Description-1")
```

Comment (leungkinghin) [reply src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java]: https://github.com/commercetools/commercetools-project-sync/pull/138/commits/c1d881d9c54051f45e61896a0dd56efeced0d1a6

**Pull request merged**